### PR TITLE
chore(*): aligned the order and spacing of fields within extension.yaml files

### PR DESCRIPTION
**Pull request by rachelsaunders**
Each extension.yaml file had a slightly different ordering and spacing for its fields.  This PR aligns this ordering and spacing across all extensions.

### CODE REVIEW

File: auth-mailchimp-sync/extension.yaml [hunk 1]
```diff
@@ -51,6 +53,7 @@ resources:
       eventTrigger:
         eventType: providers/firebase.auth/eventTypes/user.create
         resource: projects/${PROJECT_ID}
+
```
Comment (laurenzlong) [inline auth-mailchimp-sync/extension.yaml]: I'd prefer to not add line breaks between items in a list (`-` indicates items in a list). That's very non-standard for yaml files, although it does work (much to my surprise).

Comment (rachelsaunders) [reply auth-mailchimp-sync/extension.yaml]: The other extension.yaml files with multiple resource objects have line breaks.  Also, when there are multiple parameters, we also have line breaks.

In most of the yaml files used internally, we use line breaks, as well.

Comment (laurenzlong) [reply auth-mailchimp-sync/extension.yaml]: Oh interesting, I never noticed that. Ya learn something new everyday!

Comment (rachelsaunders) [reply auth-mailchimp-sync/extension.yaml]: ya, yaml files are flaky, but with this, there're not bad :-)

**Pull request merged**